app/src/main/res/values/strings.xml

```diff
@@ -141,6 +141,9 @@ Please report any problems you encounter via Github.</string>
     <!-- Setting title -->
     <string name="folder_master">Send Only</string>
 
+    <!-- Setting title -->
+    <string name="folder_fileWatcher">Realtime Sync</string>
```

Comment:
This is such a poor name, we should change it. It's nowhere close to realtime, as there is a 10 second delay.
Should probably be "Watch for changes" with a text explaining that otherwise it takes an hour to detect a change (or pressing rescan button in Web UI).

Reply:
Made https://github.com/syncthing/syncthing-android/pull/1077